Build failure: weylus

### Nixpkgs version

- Unstable (25.11)

### Steps to reproduce

`nix-build -A weylus`

### Can Hydra reproduce this build failure?

Yes, Hydra can reproduce this build failure.

### Link to Hydra build job

https://hydra.nixos.org/build/308943062

### Relevant log output

```console
error: builder for '/nix/store/ll5dfj1lrcyvrxl61ilawkl3556mdkf6-weylus-unstable-2025-02-24.drv' failed with exit code 101;
       last 25 log lines:
       >   CMAKE_x86_64_unknown_linux_gnu = None
       >   HOST_CMAKE = None
       >   CMAKE = None
       >   running: cd "/build/source/target/x86_64-unknown-linux-gnu/release/build/fltk-sys-f1a27aff9c2c02e1/out/build" && CMAKE_PREFIX_PATH="" "cmake" "/build/weylus-unstable-2025-02-24-vendor/fltk-sys-1.4.34/cfltk" "-DFLTK_USE_SYSTEM_LIBPNG=OFF" "-DFLTK_USE_SYSTEM_LIBJPEG=OFF" "-DFLTK_USE_SYSTEM_ZLIB=OFF" "-DCFLTK_LINK_IMAGES=ON" "-DOpenGL_GL_PREFERENCE=GLVND" "-DFLTK_BUILD_GL=OFF" "-DCFLTK_USE_OPENGL=OFF" "-DFLTK_USE_PANGO=ON" "-DFLTK_GRAPHICS_CAIRO=ON" "-DFLTK_BACKEND_WAYLAND=ON" "-DFLTK_USE_LIBDECOR_GTK=OFF" "-DFLTK_USE_SYSTEM_LIBDECOR=OFF" "-DCMAKE_EXPORT_COMPILE_COMMANDS=ON" "-DCFLTK_CARGO_BUILD=ON" "-DFLTK_BUILD_EXAMPLES=OFF" "-DFLTK_BUILD_TEST=OFF" "-DFLTK_BUILD_FLUID=OFF" "-DFLTK_BUILD_FLTK_OPTIONS=OFF" "-DFLTK_OPTION_LARGE_FILE=ON" "-DFLTK_BUILD_HTML_DOCS=OFF" "-DFLTK_BUILD_PDF_DOCS=OFF" "-DCMAKE_INSTALL_PREFIX=/build/source/target/x86_64-unknown-linux-gnu/release/build/fltk-sys-f1a27aff9c2c02e1/out" "-DCMAKE_C_FLAGS= -ffunction-sections -fdata-sections -fPIC -m64" "-DCMAKE_C_COMPILER=/nix/store/dmypp1h4ldn0vfk3fi6yfyf5yxp9yz0k-gcc-wrapper-14.3.0/bin/cc" "-DCMAKE_CXX_FLAGS= -ffunction-sections -fdata-sections -fPIC -m64" "-DCMAKE_CXX_COMPILER=/nix/store/dmypp1h4ldn0vfk3fi6yfyf5yxp9yz0k-gcc-wrapper-14.3.0/bin/c++" "-DCMAKE_ASM_FLAGS= -ffunction-sections -fdata-sections -fPIC -m64" "-DCMAKE_ASM_COMPILER=/nix/store/dmypp1h4ldn0vfk3fi6yfyf5yxp9yz0k-gcc-wrapper-14.3.0/bin/cc" "-DCMAKE_BUILD_TYPE=Release"
       >   -- Configuring incomplete, errors occurred!
       >
       >   --- stderr
       >   CMake Error at CMakeLists.txt:1 (cmake_minimum_required):
       >     Compatibility with CMake < 3.5 has been removed from CMake.
       >
       >     Update the VERSION argument <min> value.  Or, use the <min>...<max> syntax
       >     to tell CMake that the project requires at least <min> but has been updated
       >     to work with policies introduced by <max> or earlier.
       >
       >     Or, add -DCMAKE_POLICY_VERSION_MINIMUM=3.5 to try configuring anyway.
       >
       >
       >
       >   thread 'main' panicked at /build/weylus-unstable-2025-02-24-vendor/cmake-0.1.51/src/lib.rs:1100:5:
       >
       >   command did not execute successfully, got: exit status: 1
       >
       >   build script failed, must exit now
       >   note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
       > warning: build failed, waiting for other jobs to finish...
       For full logs, run 'nix log /nix/store/ll5dfj1lrcyvrxl61ilawkl3556mdkf6-weylus-unstable-2025-02-24.drv'.
```

### Additional context

_No response_

### System metadata

N/A

### Notify maintainers

@legendofmiracles

---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aopen+is%3Aissue+label%3A%220.kind%3A+build+failure%22). 
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/by-name/we/weylus/package.nix
+++ b/pkgs/by-name/we/weylus/package.nix
@@ -25,13 +25,13 @@
 
 rustPlatform.buildRustPackage {
   pname = "weylus";
-  version = "unstable-2025-02-24";
+  version = "unstable-2025-10-08";
 
   src = fetchFromGitHub {
     owner = "H-M-H";
     repo = "weylus";
-    rev = "5202806798ccca67c24da52ba51ee50b973b7089";
-    sha256 = "sha256-lx1ZVp5DkQiL9/vw6PAZ34Lge+K8dfEVh6vLnCUNf7M=";
+    rev = "56e29ecbde3a4aba994a9df047b5398feb447c1b";
+    hash = "sha256-dHdgWrygSXqKf9fpYRVDj+Ql97Or/kjBfN/mECy2ipc=";
   };
 
   buildInputs = [
@@ -71,7 +71,7 @@ rustPlatform.buildRustPackage {
     libtool
   ];
 
-  cargoHash = "sha256-dLhlYOrLjoBSRGDJB0qTEIb+oGnp9X+ADHddpYITdl8=";
+  cargoHash = "sha256-Mx8/zMG36qztbFYgqC7SB75bf8T0NkYQA+2Hs9/pnjk=";
 
   cargoBuildFlags = [ "--features=ffmpeg-system" ];
   cargoTestFlags = [ "--features=ffmpeg-system" ];

